critiquebrainz/frontend/templates/review/browse.html

```diff
@@ -5,6 +5,16 @@
 
 {% block content %}
 <h2>{{ _('Reviews') }}</h2>
+<form method="GET" action="{{ url_for('review.browse') }}">
+  <div class="form-group" style="display: inline-block;">
+    <select class="form-control input-sm" name="entity_type" onchange="this.form.submit()"> 
```

Comment:
I've been considering that it would be useful when we have more types of filters in the future. But if that's too much of a problem now, we can improve this later.